### DISCUSSION TITLE
Chai MSA

### DISCUSSION THREAD

**Discussion by sbpkyle** [chaidiscovery/chai-lab#142] [General]
Hi,

I wanted to ask about MSA implementation to reflect your web server. I noticed you mentioned you used shards from different databases and wanted to know 

1. if you generated full MSAs from each database first, and then just aggregated them all together or 
2. If you only partially pulled from each one and potentially used a pairing key to extract certain bits of information?

Thanks in advanced!

**Comment by jackdent** [comment]
We generate full MSAs from each database and then aggregate them
